lms/djangoapps/badges/api/views.py

```diff
@@ -0,0 +1,135 @@
+"""
+API views for badges
+"""
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from rest_framework import generics
+from rest_framework.exceptions import APIException
+
+from openedx.core.djangoapps.user_api.permissions import is_field_shared_factory
+from openedx.core.lib.api.authentication import (
+    OAuth2AuthenticationAllowInactiveUser,
+    SessionAuthenticationAllowInactiveUser
+)
+from badges.models import BadgeAssertion
+from .serializers import BadgeAssertionSerializer
```

Comment:
Maybe they are -- I just recall pep8 or pylint yelling at me about them a while back -- if it's an acceptable thing from a project standards perspective then we can leave as-is